packages/frontend/src/features/semanticViewer/components/SaveSemanticViewerChart.tsx

```diff
@@ -55,6 +64,11 @@ const SaveSemanticViewerChart: FC = () => {
         await chartUpdateMutation.mutateAsync({
             uuid: savedSemanticViewerChartUuid,
             payload: {
+                unversionedData: {
+                    name,
+                    description: savedChartQuery.data.description,
+                    spaceUuid: savedChartQuery.data.space.uuid,
+                },
```

Comment:
changes in this file fixes name update. UX is not nearly there, but at least it does not have bugs 😉